test/odo.int.ts

```diff
@@ -12,10 +12,12 @@ import { Cluster } from '../src/openshift/cluster';
 suite('odo integration', () => {
     const oi: odo.Odo = odo.getInstance();
     let sb: sinon.SinonSandbox;
+    let showInformationMessageStub: sinon.SinonStub;
 
     setup(() => {
         sb = sinon.createSandbox();
-        sb.stub(vscode.window, 'showInformationMessage').resolves('Download and install');
```

Comment:
Why replace single line with two with exactly the same result?

Reply:
after updating package.json it was giving an error for tslint that why I have replaced the single line.

Reply:
could you fix errors by configuring rules for linter?